modules/setting-up-a-customized-argocd-instance.adoc

```diff
@@ -0,0 +1,80 @@
+
+If you want to install your own instance for managing your cluster configurations or deploying your applications, you must deploy your own instance of Argo CD. Using your own instance of Argo CD helps you to continuously deliver your applications.
+
+== Installing ArgoCD
+
+To install an Argo CD instance:
+
+. In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators*.
+. Create a new project or select an existing project, depending on where you want to install the Argo CD instance.
+. In the *Installed Operators* page, select *OpenShift GitOps* and then the *Argo CD* tab.
+. Create an Argo CD instance and specify the following configurations:
+*** Name: <argocd-instance-name>
+*** *Server* -> *Route* -> *Enable Route* = true. This configuration creates an external open source (OS) route to access the Argo CD server.
+
+== Deploying resources to a different namespace
+
+To grant Argo CD the permissions to manage resources in multiple namespaces, create role bindings which provide Argo CD admin permissions for that particular namespace. Use the example, `OOTB Argo CD` instance, deployed in the `openshift-gitops` namespace.
+
+To create a role binding:
+
+. In the *Administrator* perspective of the web console, navigate to *User Management* -> *Role Bindings*, and then select *Create Binding*. You now see a form that guides you through the process of creating a role binding:
+. For *Binding Type*, select *Namespace Role Binding (RoleBinding)*. You do not need to select the binding type if a particular project is selected, as the type is assumed to be role binding for the project.
+. Under *Role Binding*, provide a role binding *Name* and *Namespace*. For the *Namespace*, specify the one you are granting permissions for.
+. In the *Role* list, select the *Cluster Role admin (CR admin)*.
+. Under *Subject*, select *Service Account* and provide the *Subject Namespace* and *Name*. For example:
+*** Subject Namespace: openshift-gitops
+*** Subject Name: argocd-cluster-argocd-application-controller
+. Select *Create* to create the role binding.
+
+== Configuring clusters
+
+After you deploy an Argo CD instance, you can manage your cluster configurations:
+
+. In the *Administrator* perspective on the web console, navigate to *Administration* -> *CustomResourceDefinition* -> *Subscription* -> *Instances*, and then select *openshift-gitops-operator*.
+. Select the `Subscription` YAML file and add:
++
+```
+spec:
+  config:
+    env:
+    - name: ARGOCD_CLUSTER_CONFIG_NAMESPACES
+      Value: openshift-gitops, <namespace where argocd instance is installed>
+```
+
+== Setting up additional permissions for Argo CD
+
+To extend the permissions for the pre-installed Argo CD instance so that it can list the secrets for all namespaces:
```

Comment:
> To extend the permissions for the pre-installed Argo CD instance so that it can list the secrets for all namespaces:

Could we indicate that this is only an example and the admin needs to _handle with care_